aws/resource_aws_ses_domain_identity_policy_test.go

```diff
@@ -0,0 +1,58 @@
+package aws
+
+import (
+	"regexp"
+	"testing"
+
+	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSSESDomainIdentityPolicy_basic(t *testing.T) {
+	domain := fmt.Sprintf(
+		"%s.terraformtesting.com.",
+		acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsSESDomainIdentityDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSSESDomainIdentityConfig_withPolicy(domain),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsSESDomainIdentityExists("aws_ses_domain_identity.test"),
```

Comment:
A new `testAccCheckAwsSESIdentityPolicyExists()` check function should be created that verifies from the API that the SES Identity Policy referenced by the given resource name was in fact properly created.